python/lsst/cp/pipe/linearity.py

```diff
@@ -106,6 +117,11 @@ class LinearitySolveConfig(pipeBase.PipelineTaskConfig,
         doc="Maximum deviation from linear solution for Poissonian noise.",
         default=5.0,
     )
+    ignorePtcMask = pexConfig.Field(
```

Comment:
Did you find cases in which it was useful to ignore the values masked by the PTC task? Also, maybe explain a bit more in the docstring, if possible, to which values you are referring to (I assume points at a certain mean signal level in ADU).

Reply:
For my test PTC, it flagged a number of points that looked reasonable between ~20k and ~50k ADU.  Even if these points don't match the expectation of the PTC code, they're still valid measurements of the flux at that particular exposure time.  Only once the detector saturates are the values no longer useful for linearity measurement.  With this option and the `maxLinearAdu` option, I was able to use all of the points less than the apparent saturation.